OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -262,10 +259,10 @@ public bool TryOpen(string filename, out Stream s)
 		public bool Exists(string filename)
 		{
 			var explicitSplit = filename.IndexOf('|');
-			if (explicitSplit > 0)
-				if (explicitMounts.TryGetValue(filename[..explicitSplit], out var explicitPackage))
-					if (explicitPackage.Contains(filename[(explicitSplit + 1)..]))
-						return true;
+			if (explicitSplit > 0 &&
```

Comment:
It would be nice if we could be consistent about whether the `&&` goes at the start or end of the line in these cases. Is there a rule for that?

Reply:
I'm not aware of one offhand, but that's not to say there isn't one somewhere.

I've generally wrapped after as I _think_ that's the more common convention in the codebase, but even in this chageset I wrap before in `FieldLoader` as that's the more common convention in that file.

Reply:
I usually find the `&&` or `||` nicer and more readable when placed at the start. Though it does depend on the length of the first line